docs/backups.md

```diff
@@ -338,3 +337,32 @@ When a backup job is configured using Normal snapshot mode, it's possible to use
 - **xo-memory-backup** to apply RAM-enabled snapshotting
 
 For example, you could have a regular backup job with 10 VMs configured with Normal snapshotting, including two which are database servers. Since database servers are generally more sensitive to being restored from snapshots, you could apply the **xo-memory-backup** tag to those two VMs and only those will be backed up in RAM-enabled mode. This will avoid the need to manage a separate backup job and schedule.
+
+## Retention and Scheduling
+
+Just a refresher/summary: You can select multiple backup methods for the same job:
+
+- Full: 'Backup' and 'Disaster Recovery' (DR)
```

Comment:
Use `*word*` to emphasize a word in Markdown, you can also use backticks to use code spans (e.g. `` `word` ``), which is useful to indicate a piece of code or that a text is copied verbatim.